app/src/main/res/drawable/ic_baseline_settings_24.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
```

Comment:
maybe create a dim. field for 24 in values xml?

Reply:
Yeah good idea